Makefile.tools

```diff
@@ -0,0 +1,146 @@
+SHELL=/usr/bin/env bash -o pipefail
+
+## Tools
+TOOLS_DIR = $(shell pwd)/tmp/bin
+
+## NOTE: each tool must have a version that will be recorded in .github/tools
+# The .github/tools file's hash is used to compute the key for cache in github
+# see: .github/tools-cache/action.yaml
+
+CONTROLLER_GEN=$(TOOLS_DIR)/controller-gen
+CONTROLLER_GEN_VERSION= v0.7.0
+
+KUSTOMIZE=$(TOOLS_DIR)/kustomize
+KUSTOMIZE_VERSION= v3.9.4
+
+OPERATOR_SDK = $(TOOLS_DIR)/operator-sdk
+OPERATOR_SDK_VERSION = v1.13.0
+
+OPM=$(TOOLS_DIR)/opm
+OPM_VERSION = v1.15.1
+
+GOLANGCI_LINT=$(TOOLS_DIR)/golangci-lint
+GOLANGCI_LINT_VERSION= v1.42.1
+
+## NOTE: promq does not have any releases, so we use a fake version starting with v0.0.1
+# thus to upgrade/invalidate the github cache, increment the value
+PROMQ = $(TOOLS_DIR)/promq
+PROMQ_VERSION = v0.0.1
+
+# NOTE: oc is NOT downloadable using the OC_VERSION in its URL, so this has to be manually updated
+OC = $(TOOLS_DIR)/oc
+OC_VERSION = v4.9.8
+
+CRDOC = $(TOOLS_DIR)/crdoc
```

Comment:
could you please modify the tools/cache action to validate / print the `crdoc` version

Reply:
Can you ptal again? I moved this part to the Makefile so that it's more concise and we know that we need to update it when we add/remove tools

Reply:
Sorry, I missed this update. Yes, it looks good to me but needs rebasing :(